src/core/cell.js

```diff
@@ -201,6 +201,7 @@ const evalSuffixExpr = (srcStack, formulaMap, cellRender, cellList) => {
     }
     // console.log('stack:', stack);
   }
+  stack[0] = Number(stack[0]).toFixed(2).toString();
```

Comment:
Only round if computation includes subtraction and results in a value more than 5 decimals